scripts/remove_boilerplate.py

```diff
@@ -191,11 +204,36 @@ def parse_arguments():
     return args
 
 
+def good_token(token):
+    # A token is good if:
+    # * It is on the whitelist
+    # * It is a word ending in ...
+    # * It does not have 3 consecutive punctuation marks
+    global whitelist
+    if token in whitelist:
+        return True
+    matcher3dots = re.compile(r'^\w+\.\.\.$')
+    if matcher3dots.match(token):
+        return True
+    matcher3punct = re.compile(r'.*[^\w\s]{3,}')
+    if matcher3punct.match(token):
+        return False
+    else:
```

Comment:
I would prefer this without the `else`. Since you are not doing nested `if`s anyway, let's just return in the main branch and let the `if`s be side-branches.

Reply:
Ok.